src/graphnet/models/cnn/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""CNN-specific modules, for performing the main learnable operations."""
+
+from .cnn import CNN
+from .theos_muonE_upgoing import TheosMuonEUpgoing
```

Comment:
`.theos_muonE_upgoing` breaks with snake-case convention. Do we need "theos" in there? It's very jargony